plugins/poll/app/models/poll.rb

```diff
@@ -40,7 +41,7 @@ def is_closed?
   end
 
   def can_see_results?(user)
-    always? || is_closed? || (on_vote? && has_voted?(user))
+    always? || is_closed? || (on_vote? && has_voted?(user)) || (staff_only? && user&.staff?)
```

Comment:
If we only ever want staff member to see the results, we might want to rewrite this method like this

``` ruby
def can_see_results?(user)
  return true if always?
  return user&.staff? if staff_only?
  return has_voted?(user) if on_vote?
  is_closed?
end
```